src/main/java/org/purescript/parser/PSElements.kt

```diff
@@ -86,6 +86,26 @@ interface PSElements {
         @kotlin.jvm.JvmField
         val ImportDeclaration = PSElementType("ImportDeclaration")
         @kotlin.jvm.JvmField
+        val ImportAlias = PSElementType("ImportAlias")
+        @kotlin.jvm.JvmField
+        val ImportList = PSElementType("ImportList")
+        @kotlin.jvm.JvmField
+        val ImportedClass = PSElementType("ImportedClass")
+        @kotlin.jvm.JvmField
+        val ImportedData = PSElementType("ImportedData")
+        @kotlin.jvm.JvmField
+        val ImportedDataMemberList = PSElementType("ImportedDataMemberList")
+        @kotlin.jvm.JvmField
+        val ImportedDataMember = PSElementType("ImportedDataMember")
```

Comment:
The compiler reuses its data member types for both imports and exports, but since the references are resolved differently I think it makes sense for us to keep them separate.